### PR TITLE
Announce WildFly bootable JAR 2.0 Final

### PR DESCRIPTION
**Pull request by jfdenise**
@bstansberry that is the blog post to announce Bootable JAR 2.0

### CODE REVIEW

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+shows how to build a cluster of bootable JARs running on OpenShift.
+
+The details on how to configure a bootable JAR for cloud execution can be found in the 
+link:https://docs.wildfly.org/bootablejar/#wildfly_jar_configuring_cloud[Configuring the server for cloud execution] chapter of the Maven plugin documentation.
+
+#### Zero configuration with Eclipse JKube plugin
+
+The link:https://www.eclipse.org/jkube/[JKube] plugin automates build and deployment 
+of your application for OpenShift or Kubernetes. 
+By combining WildFly bootable JAR and JKube plugins you can achieve "zero configuration"
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:205]: s/combining/combining the/g

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 2]
```diff
@@ -0,0 +1,367 @@
...
+        </enricher>
+      </configuration>
+    </plugin>
+  </plugins>
+</build>
+----
+
+Then call ```mvn oc:deploy``` to build your deployment, create a bootable JAR and deploy it onto openshift.
+
+IMPORTANT: Make sure you are logged in to your OpenShift Cluster prior to call ```oc:deploy```.
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:258]: s/call/calling

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 3]
```diff
@@ -0,0 +1,367 @@
...
+</configuration>
+----
+
+The link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Final/examples/https[https example] 
+shows how to package extra content. 
+
+
+## Testing with Arquillian
+
+link:https://github.com/wildfly/wildfly-arquillian[WildFly Arquillian] 3.0 contains a support for WildFly bootable JAR
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:344]: s/a support/support/g
s/for WildFly/for a WildFly/g

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 4]
```diff
@@ -0,0 +1,367 @@
...
+
+
+## Testing with Arquillian
+
+link:https://github.com/wildfly/wildfly-arquillian[WildFly Arquillian] 3.0 contains a support for WildFly bootable JAR 
+allowing you to test deployments with Arquillian.
+
+The ```org.wildfly.arquillian:wildfly-arquillian-container-bootable``` adapter for arquillian supports:
+
+* Execution of tests that run as client against a running bootable JAR.
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:349]: s/client/a client/g

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 5]
```diff
@@ -0,0 +1,367 @@
...
+* Execution of tests that run inside a running bootable JAR but with some limitations. 
+The bootable JAR to test must be an Hollow JAR (a JAR that doesn't contain any deployment).
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:351]: s/an Hollow/a Hollow/g

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 6]
```diff
@@ -0,0 +1,367 @@
...
+allowing you to test deployments with Arquillian.
+
+The ```org.wildfly.arquillian:wildfly-arquillian-container-bootable``` adapter for arquillian supports:
+
+* Execution of tests that run as client against a running bootable JAR.
+* Execution of tests that run inside a running bootable JAR but with some limitations. 
+The bootable JAR to test must be an Hollow JAR (a JAR that doesn't contain any deployment). 
+There is no support yet to repackage an existing deployment to contain the classes of the deployment created by the test.
+
+The https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Final/examples/arquillian[arquillian examples] shows how to
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:354]: s/shows/show

File: _posts/2020-10-19-bootable-jar-2.0-released.adoc [hunk 7]
```diff
@@ -0,0 +1,367 @@
...
+* Execution of tests that run inside a running bootable JAR but with some limitations. 
+The bootable JAR to test must be an Hollow JAR (a JAR that doesn't contain any deployment). 
+There is no support yet to repackage an existing deployment to contain the classes of the deployment created by the test.
+
+The https://github.com/wildfly-extras/wildfly-jar-maven-plugin/tree/2.0.0.Final/examples/arquillian[arquillian examples] shows how to 
+use WildFly arquilian to test deployments running inside a bootable JAR.  
+
+## To conclude
+
+We hope that you are seeing an interest in this new WildFly application packaging,
```
Comment (bstansberry) [inline _posts/2020-10-19-bootable-jar-2.0-released.adoc:359]: s/packaging,/packaging;/g

**Review by bstansberry — CHANGES_REQUESTED** [review]
Very nice. I've got a bunch of minor comments, but the content looks good.

**Comment by jfdenise** [comment]
@bstansberry , thank-you, force-pushed.

**Review by bstansberry — APPROVED** [review]

**Pull request merged**